demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -62,6 +64,39 @@ def create_table(expected: Dict[str, Any], received: Dict[str, Any]) -> Table:
     return table
 
 
+# https://stackoverflow.com/questions/5891555/display-the-date-like-may-5th-using-pythons-strftime#answer-5891598
```

Comment:
It's better to add docstring to the function explaining what it does instead of a link to stackoverflow